packages/truffle-core/lib/commands/develop.js

```diff
@@ -83,7 +83,7 @@ var command = {
 
     var ganacheOptions = {
       host: "127.0.0.1",
-      port: 9545,
+      port: 8545,
```

Comment:
Can we do this! If Ganache is already running, then `truffle develop` will fail to start, no?

Reply:
It would be prudent to error out and let the user know something like "Looks like you already have Ganache running on port 8545. Either change the port or turn off Ganache and try the command again."

Reply:
The whole point of `truffle develop` is that it's low friction. That's why we chose 9545 for `truffle develop` in the first place and 7545 for Ganache UI

Reply:
@gnidan in v5/next/develop the current behavior is that `truffle develop` doesn't clash (out of the box) with ganache-cli or ganacheUI.

on this branch/PR, `truffle develop` doesn't clash with ganacheUI out of the box, but if `ganache-cli` is already running and you try to spin up `truffle develop`, it'll clash and throw an error (because of the timestamped network_id's ganache-cli is given).  However, if you spin up `truffle develop` and then start up `ganache-cli`, truffle will auto-connect to `ganache-cli`.